client/src/components/AnnouncementCard.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+export default function AnnouncementCard() {
+  const [isShown, setIsShown] = useState(false);
+  return (
+    <div>
+      <div
+        class="block p-6 shadow-lg bg-white w-full border-b-2"
+        onMouseEnter={() => setIsShown(true)}
+        onMouseLeave={() => setIsShown(false)}
+        style={{
+          backgroundColor: isShown ? "#EAE8E8" : "",
+        }}
+      >
+        <div className="flex">
+          <img
+            src="https://mdbcdn.b-cdn.net/img/new/avatars/2.webp"
+            className="rounded-full w-12"
+            alt="Avatar"
+          />
+          <h5 class="leading-tight font-bold ml-3 mb-2">Admin</h5>
+          <p class="ml-2 text-xs mt-[2px]">Sept. 21, 2022 at 12:44PM</p>
+          {isShown && (
+            <div className="ml-auto">
```

Comment:
- [ ] add a background color like in Figma
- [ ] but since the bg-color of the card is already gray, set bg-color to white instead
```
{isShown && <div className="drop-shadow-md px-2 py-0.5 bg-white text-gray-500 border-gray-500 border-2 rounded cursor-pointer">
  <svg
    xmlns="http://www.w3.org/2000/svg"
    fill="none"
    viewBox="0 0 24 24"
    strokeWidth={1.5}
    stroke="currentColor"
    className="w-6 h-6"
  >
    <path
      strokeLinecap="round"
      strokeLinejoin="round"
      d="M20.25 8.511c.884.284 1.5 1.128 1.5 2.097v4.286c0 1.136-.847 2.1-1.98 2.193-.34.027-.68.052-1.02.072v3.091l-3-3c-1.354 0-2.694-.055-4.02-.163a2.115 2.115 0 01-.825-.242m9.345-8.334a2.126 2.126 0 00-.476-.095 48.64 48.64 0 00-8.048 0c-1.131.094-1.976 1.057-1.976 2.192v4.286c0 .837.46 1.58 1.155 1.951m9.345-8.334V6.637c0-1.621-1.152-3.026-2.76-3.235A48.455 48.455 0 0011.25 3c-2.115 0-4.198.137-6.24.402-1.608.209-2.76 1.614-2.76 3.235v6.226c0 1.621 1.152 3.026 2.76 3.235.577.075 1.157.14 1.74.194V21l4.155-4.155"
    />
  </svg>
</div>}
```

- [ ] you can also add `absolute top-0 right-0 translate-y-1/2 -translate-x-1/2` and set the parent to `relative` to set the position of the thread icon